src/test/resources/application.properties

```diff
@@ -1,19 +1,17 @@
-#spring.main.banner-mode=off
+spring.main.banner-mode=off
```

Comment:
The only things needed in test/resources/application.properties are the values that are different from main/resources/application.properties

By default spring, while executing tests, spring will treat test properties as higher priority than main properties but it does read both.  By adding all properties into test you risk divergence between configuration of main and test that might not be intended.

Reply:
Implemented.